### PR TITLE
Fixes #19476: Make rudder package command use version files

### PR DESCRIPTION
**Pull request by peckpeck**
https://issues.rudder.io/issues/19476

### CODE REVIEW

File: relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py [hunk 1]
```diff
@@ -625,23 +625,34 @@ def list_plugin_name():
...
+                step += m.group(2)
+            m = re.match(r'main_version=7.0.0 *= *((\d+\.\d+)\.\d+)', line)
+            if m:
+                RUDDER_MAJOR = m.group(2)
+                rudder_version = m.group(1)
+        RUDDER_VERSION = rudder_version + step
 except:
     print(
-        'Warning, cannot retrieve major Rudder version ! Verify that rudder is well installed on the system'
+        'Warning, cannot retrieve Rudder version ! Verify that rudder is well installed on the system'
```
Comment (ncharles) [inline relay/sources/rudder-pkg/lib/rudder-pkg/rudderPkgUtils.py:655]:
```suggestion
        'Warning, cannot retrieve the Rudder version! Verify that Rudder is correctly installed on this system'
```

**Comment by peckpeck** [comment]
PR updated with a new commit

**Comment by amousset** [comment]
Format tests failing, you can run `make fmt` in `rudder/relay/sources`

**Comment by peckpeck** [comment]
PR updated with a new commit

**Review by amousset — APPROVED** [review]

**Pull request merged**